packages/rocketchat-lib/server/functions/checkUsernameAvailability.js

```diff
@@ -2,12 +2,12 @@ import { Meteor } from 'meteor/meteor';
 import s from 'underscore.string';
 
 
-let usernameBlackList = [];
+let usernameBlackList = ['all', 'here'];
 
 const toRegExp = (username) => new RegExp(`^${ s.escapeRegExp(username).trim() }$`, 'i');
 
 RocketChat.settings.get('Accounts_BlockedUsernameList', (key, value) => {
-	usernameBlackList = value.split(',').map(toRegExp);
+	usernameBlackList = usernameBlackList.concat(value.split(',').map(toRegExp));
```

Comment:
This isn't a good solution, it will keep adding more itens to the array every time this setting is updated.

Reply:
hi
thank for your replay ! 
i have changed it